examples/telegraf/batch_template.tick

```diff
@@ -0,0 +1,46 @@
+// {alert_name}
```

Comment:
These are templates for your examples and I like it. Do you think naming this  a `batch_template` will later confuse this script with actual TICKscript templates?

Maybe we should call these generic examples or something? Or maybe convert them to real templates? Thoughts?